storage/lrucache/capacity/capacityLRUCache.go

```diff
@@ -0,0 +1,252 @@
+package capacity
+
+import (
+	"container/list"
+	"fmt"
+	"sync"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+var log = logger.GetOrCreate("storage/lrucache/capacity")
+
+// CapacityLRU implements a non thread safe LRU Cache with a max capacity size
+type CapacityLRU struct {
```

Comment:
is it must for this to be public ?

Reply:
no, will switch both implementation to unexported type

Reply:
done